spec/modules/facts/functions/group_by_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'facts::group_by' do
```

Comment:
I think we can safely test this in the module since it doesn't actually run tasks